src/more-information.md

```diff
@@ -0,0 +1,6 @@
+# Other places to find information
+
+You are going to find [Rustc API docs site] useful and there's more supplementary documentation in [Rust Forge].
```

Comment:
should we put it in a separate file? my only concern is that then it's a bit less obvious. Maybe it should just be part of `about-this-guide.md`?

Reply:
Fixed.